### DISCUSSION TITLE
Rework sur les attaques du canonnier

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#1306] [Nouvelles idées]
Autre retour concernant cette màj, j'ai récemment discuté avec un groupe de joueurs afin de voir de possibles alternatives au gameplay du canonnier, il en est sorti que :
- l'attaque troublante devrait être retravaillée car elle est beaucoup trop aléatoire, nous avons donc pensé limiter cette attaque sur un ou deux tours mais avec 100% de chance de réussites, c'est à dire 100% de chances de soit bloquer, soit faire réaliser une attaque aléatoire (la répartition du pourcentage des deux effets serait encore à définir, 50% de chances de bloquer serait encore trop excessif) 
- l'attaque fourbe se révèle être très peu utile en combat en raison de l'incapacité des classes axées sur l'attaque à se faire de réels dégâts et n'est donc réalisable qu'au premier tour, nous pensons donc qu'envisager de la remplacer par une nouvelle attaque pourrait être une bonne idée, cela contribuerait notamment à la diversification du moveset purement offensif (manque de boost personnel) 
- l'attaque canon est carrément évitée en fight car son taux d'échec important et le fait que la quantité de dégâts infligés est absolument le même qu'une attaque rapide ou sabotage (oui d'ailleurs niveau dégâts peu importe l'attaque elles ne se différencient pas vraiment en puissance) la rend absolument désavantageuse, elle devrait donc pouvoir faire bien plus dégâts en cas de réussite (quitte à diminuer encore sa proba/en faire une attaque similaire à la divine)

**Comment by BastLast** [comment]
Je vais être honnête, je pense que je vais couper la discussion pour le moment pour 2 raisons :

- jai très très peu de temps a accorder à draftbot pour les prochaines semaines, du coup jai pas le temps de bosser sur les fights

- on va sortir une maj qui va nous permettre de faire pleins de stats qui nous permettront detre bcp bcp plus rigoureux, juste et efficace dans ce genre de trucs et de pas juste faire au ressenti.

Du coup je met en hold pour au moins jusqu'à la prochaine grosse maj histoire qu'on gaspille pas nos efforts.
